### ISSUE TITLE
Adding an owner to an org fails for Circles accounts with more than one owner

### ISSUE THREAD

**Issue by llunaCreixent** [CirclesUBI/circles-myxogastria#257]
There are conflicts in the database and only one owner is added, presumably because if the owner additions are executed in parallel the same nonce is obtained from the org safe contract. Then only the first tx is valid and the rest produce this error:
```
2021-11-10 08:36:22,552 [WARNING] [MainProcess] https://relay.circles.garden/api/v1/safes/0x03510596636630a5392e63D58Fd81989839C8A10/transactions/ - Exception: SafeMultisigTxExists: Tx with safe-nonce=2 for safe=0x03510596636630a5392e63D58Fd81989839C8A10 already exists in DB - Data received {...}
```

Check the code:
https://github.com/CirclesUBI/circles-myxogastria/blob/9d36f18186cde8f1e92a3c25583d35efee3a77b0/src/store/safe/actions.js#L354

**Comment by triaslucia** [comment]
This looks like it affects user in very particular scenarios. Check it to understand priority.

**Comment by llunaCreixent** [comment]
One example: I have added one owner to my Safe through the gnosis app (https://app.safe.global/settings/setup?safe=gno%<safe-address>). Thus you see two entries in your screenshoot. You can also check the owners of a safe asking the subgraph. Then for each owner I ask for the Safes, And I obtain the amount of them that are not an org:
```
{
  user(id: "...") {
    safes {
      id
      organization
    }
  }
}
```

So for the users who have organizations that were not fully deployed they will have several avatars in their "members" entry.
And having two entries is associated to users that have added an owner through another app like gnosis app.

**Comment by louilinn** [comment]
Yes so it is confirmed that it affects very few users and it could probably be fixed by removing other owners of a personal wallet to restore the functionality of being able to be added to a shared wallet. 

Context, when adding a shared wallet member we:
- search for personal wallets and chose one
- find the owner of the personal
- the owner of the personal wallet is added as an owner to the shared wallet

I think it is ok for now that we only allow in the app to add users (owners of personal wallets) as an owner of a shared wallet if they have a single clear owner, rather than adding all there owners in case they have several.

If we decide to prioritize fixing this later we can chose between
- adding one of the owners when there are several by random or by other rule
- adding all owners of the chose personal wallet

**Comment by llunaCreixent** [comment]
I think the ideal solution is that the app detects the different owners of the individual wallet candidate to be member of the SW, and lets the user decide which owners (device address) to add. So I imagine the avatars of the Safe with a text indicating some of the caracters of the device address. By default the app would add all the owners (i think this is what happens now).

We have to check how to do it but ideally the members of the SW are the individual wallet addresses... but this is at contract level and the tx ñs have to be signed by a true owner of the Safe. We could investigate more this option...
